packages/simpleserialize.com/src/util/types.ts

```diff
@@ -28,13 +28,15 @@ phase0 = patchSszTypes(phase0);
 altair = patchSszTypes(altair);
 bellatrix = patchSszTypes(bellatrix);
 capella = patchSszTypes(capella);
+deneb = patchSszTypes(deneb);
 primitive = patchSszTypes(primitive);
```

Comment:
need to make sure primitive types are not polluted, have you checked that it does not contain extra imports?

https://github.com/ChainSafe/ssz/blob/9cf6991e7ece4e4002668f601ecb43bb8bd53f4e/packages/simpleserialize.com/src/util/types.ts#L22-L25

Reply:
> need to make sure primitive types are not polluted, have you checked that it does not contain extra imports?
> 
> https://github.com/ChainSafe/ssz/blob/9cf6991e7ece4e4002668f601ecb43bb8bd53f4e/packages/simpleserialize.com/src/util/types.ts#L22-L25

Good question, I didn't check that. Looks like there isn't any name collision between `deneb` and `primitive`.

master branch `primitive`
```
0 "BLSPubkey"
1 "BLSSignature"
2 "Boolean"
3 "Byte"
4 "Bytes20"
5 "Bytes32"
6 "Bytes4"
7 "Bytes48"
8 "Bytes8"
9 "Bytes96"
10 "CommitteeIndex"
11 "Domain"
12 "DomainType"
13 "Epoch"
14 "EpochInf"
15 "ExecutionAddress"
16 "ForkDigest"
17 "Gwei"
18 "ParticipationFlags"
19 "Root"
20 "Slot"
21 "SubcommitteeIndex"
22 "SyncPeriod"
23 "Uint16"
24 "Uint32"
25 "Uint8"
26 "UintBn128"
27 "UintBn256"
28 "UintBn64"
29 "UintNum64"
30 "UintNumInf64"
31 "ValidatorIndex"
32 "Version"
33 "Wei"
34 "WithdrawalIndex"
```

nc/deneb-fork-gui `primitive`
```
0 "BLSPubkey"
1 "BLSSignature"
2 "BlobIndex"
3 "Boolean"
4 "Byte"
5 "Bytes20"
6 "Bytes32"
7 "Bytes4"
8 "Bytes48"
9 "Bytes8"
10 "Bytes96"
11 "CommitteeIndex"
12 "Domain"
13 "DomainType"
14 "Epoch"
15 "EpochInf"
16 "ExecutionAddress"
17 "ForkDigest"
18 "Gwei"
19 "ParticipationFlags"
20 "Root"
21 "Slot"
22 "SubcommitteeIndex"
23 "SyncPeriod"
24 "Uint16"
25 "Uint32"
26 "Uint8"
27 "UintBn128"
28 "UintBn256"
29 "UintBn64"
30 "UintNum64"
31 "UintNumInf64"
32 "ValidatorIndex"
33 "Version"
34 "Wei"
35 "WithdrawalIndex"
```